package.json

```diff
@@ -65,4 +65,4 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
   }
-}
\ No newline at end of file
+}
```

Comment:
`package*` file changes are not needed, please review - always best to only commit the changes you want not everything

Reply:
Alright noted 👍 let me correct that